### PR TITLE
install.sh: fix ambiguous instructions for modifying PATH

### PR DESCRIPTION
**Pull request by alebcay**
Concern raised from https://github.com/withfig/fig/discussions/300. Opening this as a starting point for making this instruction more clear.

### CODE REVIEW

**Comment by carlocab** [comment]
I was just about to open a PR for this...

**Comment by alebcay** [comment]
Well I'm open to your feedback (and from others) on this wording also 😄

File: install.sh [hunk 1]
```diff
@@ -750,7 +750,7 @@ case "$SHELL" in
 esac
 if [[ "$UNAME_MACHINE" == "arm64" ]] || [[ -n "${HOMEBREW_ON_LINUX-}" ]]; then
   cat <<EOS
-- Add Homebrew to your ${tty_bold}PATH${tty_reset} in ${tty_underline}${shell_profile}${tty_reset}:
+- Run these commands to add Homebrew to your ${tty_bold}PATH${tty_reset} via ${tty_underline}${shell_profile}${tty_reset}:
```
Comment (carlocab) [inline install.sh:753]:
```suggestion
- Run these two commands to add Homebrew to your ${tty_bold}PATH${tty_reset} via ${tty_underline}${shell_profile}${tty_reset}:
```
Let's try to remove all ambiguity here...

File: install.sh [hunk 2]
```diff
@@ -750,7 +750,7 @@ case "$SHELL" in
 esac
 if [[ "$UNAME_MACHINE" == "arm64" ]] || [[ -n "${HOMEBREW_ON_LINUX-}" ]]; then
   cat <<EOS
-- Add Homebrew to your ${tty_bold}PATH${tty_reset} in ${tty_underline}${shell_profile}${tty_reset}:
+- Run these two commands to add Homebrew to your ${tty_bold}PATH${tty_reset} via ${tty_underline}${shell_profile}${tty_reset}:
```
Comment (XuehaiPan) [inline install.sh:753]: PATH via profile -> PATH in profile via

Comment (alebcay) [reply install.sh:753]: Hm, what about just removing the `${shell_profile}` part entirely? I feel like including that in any way might still confuse some people.

Comment (carlocab) [reply install.sh:753]: @alebcay sounds good to me. We can also make it
```suggestion
- Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
```

Comment (XuehaiPan) [reply install.sh:753]: Same modification at line 763:

https://github.com/Homebrew/install/blob/61f40dfdd5f904122f7cfbc2d4c91b31290d4608/install.sh#L763-L765

Comment (gromgit) [reply install.sh:753]: How about just `Run these commands to complete your Homebrew installation`?

Comment (alebcay) [reply install.sh:753]: Good catch for additional modification and thanks also for that wording suggestion. Adding both of those now.

Comment (alebcay) [reply install.sh:753]: @XuehaiPan for that additional modification, is it a constant number of commands (e.g. 2)? Or variable number? I'm not as familiar with the syntax used here.

Comment (carlocab) [reply install.sh:753]: @gromgit I think I draw the line at "being aware that `PATH` is a thing used in the terminal" as the minimum level of literacy here 😄

Comment (XuehaiPan) [reply install.sh:753]: > is it a constant number of commands (e.g. 2)
It's 2 (for set homebrew/brew or homebrew/brew) or 4 (for set both).

**Comment by comex** [comment]
What about the possibility of not printing the “append to profile” command at all, and instead just giving the user a yes/no prompt for whether the script itself should append it to your profile?

I guess one benefit of printing the command is allowing the user to customize it if they want, e.g. by changing which file it’s appended to.  But this seems like a rare use case.  Users who are actively curating their shell profile would likely want to open it in an editor and find the best place to insert the line, rather than blindly append to any file.  And users who aren’t actively curating their shell profile would likely stick with the default choice of file.

Perhaps there’s some other benefit I’m missing.

**Comment by carlocab** [comment]
Thanks for the suggestion, @comex, but prompts are a non-starter for us. We don't do prompts to maximise compatibility with scripting and non-interactive use (e.g. in a CI run). (There are a handful of `sudo` exceptions -- one here, and a few when installing Casks or using `brew services`, but we do our best to keep those to the absolute minimum.)

We could maybe add more code to detect whether this is being used in an interactive context where we can expect a response from the user, but 1) those tests can get unreliable, and 2) that's just more code that requires more maintenance. Let's just keep things simple, shall we?

File: install.sh (hunk 2 above)
Comment (MikeMcQuaid) [reply install.sh:753]: > - Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:

I like this 👍🏻

**Review by MikeMcQuaid — COMMENTED** [review]
Nice work here!

File: install.sh (hunk 2 above)
Comment (carlocab) [inline install.sh:753]: Actually, let's make this super explicit:
```suggestion
- Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
```

**Comment by kristofferR** [comment]
Why not simply add a semicolon on the end of the first command? 

That way it'll still work even as copied as a single command.

**Comment by alebcay** [comment]
For some reason I can only re-request review from Mike, but @carlocab, @XuehaiPan, @gromgit if you all would take another look please.

**Review by carlocab — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by gromgit** [comment]
Looks good, thanks @alebcay!

**Comment by XuehaiPan** [comment]
The modification looks good for this issue. But I found an old issue for the CI test from the GitHub action result.

Screenshot

The non-default remote configuration is not tested. These two line:

https://github.com/Homebrew/install/blob/61f40dfdd5f904122f7cfbc2d4c91b31290d4608/.github/workflows/tests.yml#L95-L96

maybe need to change to (% -> #):

```
export HOMEBREW_BREW_GIT_REMOTE="${HOMEBREW_BREW_DEFAULT_GIT_REMOTE/#https/git}"
export HOMEBREW_CORE_GIT_REMOTE="${HOMEBREW_CORE_DEFAULT_GIT_REMOTE/#https/git}"
```

**Comment by carlocab** [comment]
@XuehaiPan you can probably fix that in a separate PR.

**Comment by MikeMcQuaid** [comment]
Thanks @alebcay!

**Pull request merged**